testing/cli/test_cli_sct_process_segmentation.py

```diff
@@ -96,6 +97,38 @@ def test_sct_process_segmentation_check_normalize_missing_predictor(dummy_3d_mas
         assert e.value.code == 2
 
 
+def test_sct_process_segmentation_check_normalize_PAM50(tmp_path):
+    """ Run sct_process_segmentation with -normalize PAM50"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    sct_process_segmentation.main(argv=['-i', sct_test_path('t2', 't2_seg-manual.nii.gz'), '-normalize-PAM50', '1',
+                                        '-perslice', '1', '-vertfile', sct_test_path('t2', 't2_seg-manual_labeled.nii.gz'), '-o', filename])
+    with open(filename, "r") as csvfile:
+        reader = csv.DictReader(csvfile, delimiter=',')
+        rows = list(reader)
+        row = rows[827]
+        assert row['Slice (I->S)'] == '827'
+        assert float(row['MEAN(area)']) == pytest.approx(71.96880493869594)
+        assert row['VertLevel'] == '5'
```

Comment:
It looks like with the new changes, this test is now failing:

```
_____________ test_sct_process_segmentation_check_normalize_PAM50 ______________
Traceback (most recent call last):
  File "/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/testing/cli/test_cli_sct_process_segmentation.py", line 108, in test_sct_process_segmentation_check_normalize_PAM50
    row = rows[827]
IndexError: list index out of range
```

With the smaller CSV files, we need to pick a new row to check. :stuck_out_tongue_closed_eyes:

Reply:
changed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/660581eae89cd0cb454988de0bc07addf5fa3c58